src/meta/DefMeta.jsx

```diff
@@ -1,8 +1,33 @@
 import { Helmet } from "react-helmet";
+import { useState, useEffect } from "react";
+import { useLocation } from "react-router-dom";
 
 function DefMeta() {
+  const [title, setTitle] = useState("ReactPlay - One app to learn, create, and share ReactJS projects")
+  const currentPath = useLocation().pathname
+
+  useEffect(() => {
+    if (currentPath === "/plays") {
```

Comment:
It works but I think we can simplify the code a bit here.

How about a simple JSON map with key is the current path and and value the title. We can have it managed outside of this file altogether... so passing the current path we get the title, and if it is undefined, we set the default title.

This way, tomorrow a new path comes, this file no need to go through changes for it. What say?

Reply:
That's a great idea, didn't think of that. I'll make the changes and update you.

Reply:
@atapas, for this simplification, I'm thinking of an array that holds the path and it's corresponding title. Then we check if the current path is in the this array and then assign the relevant title, if the path is not defined then we set a default title. So if we add a new page in the future, all we have to do is add the path and title for it inside the array. Is this what you have suggested? Let me know your final thoughts.